example/cumulus-tf/orca.tf

```diff
@@ -23,7 +23,6 @@ module "orca" {
   lambda_subnet_ids        = local.subnet_ids
   permissions_boundary_arn = var.permissions_boundary_arn
   prefix                   = var.prefix
-  rds_security_group_id    = local.rds_security_group
```

Comment:
Isn't this still a Cumulus variable? Why was it moved to ORCA variables?

Reply:
I followed the orca document: https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-with-cumulus